packages/clerk-js/src/ui/components/OrganizationProfile/VerifyDomainPage.tsx

```diff
@@ -98,8 +108,12 @@ export const VerifyDomainPage = withCardStateProvider(() => {
 
   const onSubmitPrepare = (e: React.FormEvent) => {
     e.preventDefault();
+    if (!domain) {
+      return;
+    }
+    affiliationEmailAddressRef.current = `${emailField.value}@${domain.name}`;
```

Comment:
💡 I think we should move this email address string to a separate function so that `${emailField.value}@${domain.name}` string exists only once.

Reply:
I think just a variable will do